src/app.hooks.js

```diff
@@ -152,8 +152,8 @@ function setupAppHooks(app) {
 
 	const after = {
 		all: [],
-		find: [],
-		get: [],
+		find: [sanitizeDataHook],
```

Comment:
This is called by every intern and extern request in der API. I will prefere that it is only execute for outgoing results.
iff(isProvider('external'), sanitizeDataHook)

But i am not happy with this solution, it will cost many many resources and performance over all. 
@janrenz 
I think this is your decision.

Reply:
Just let me know whether to leave it or discard this pull request. Currently there are some test failing because of those 'after' hooks (see image below -> exceed max deep) and I am fighting with them.
![image](https://user-images.githubusercontent.com/5371568/81167907-53156980-8f96-11ea-8df0-6bb0564bc835.png)

Reply:
This sees wrong. :)

Reply:
Jan say for the moment it is ok, but it should switch to extern request only. 

See iff(isProvider('external'), sanitizeDataHook).

Reply:
done :+1: